route_following_plugin/package.xml

```diff
@@ -18,20 +18,36 @@
 
 <package format="3">
   <name>route_following_plugin</name>
-  <version>3.2.0</version>
+  <version>4.0.0</version>
   <description>The strategic plugin generates maneuvers to make the vehicle follow the route. </description>
   <maintainer email="carma@dot.gov">carma</maintainer>
   <license>Apache License 2.0</license>
   <author email="carma@dot.gov">carma</author>
-  <buildtool_depend>catkin</buildtool_depend>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
   <build_depend>carma_cmake_common</build_depend>
-  <depend>roscpp</depend>
+  <build_depend>ament_auto_cmake</build_depend>
+
+  <depend>rclcpp</depend>
+  <depend>carma_ros2_utils</depend>
+  <depend>rclcpp_components</depend>
   <depend>std_msgs</depend>
-  <depend>cav_srvs</depend>
-  <depend>cav_msgs</depend>
-  <depend>carma_utils</depend>
-  <depend>carma_wm</depend>
+  <depend>carma_wm_ros2</depend>
   <depend>tf</depend>
```

Comment:
I could be wrong, but we no longer need tf in ROS2. It was not compiling with it I think. You can change any object/class accessing `tf` in the code to `tf2`, in which you can also find the equivalent function that was being used.